assets/js/Containers/FrevaGPT/index.js

```diff
@@ -329,58 +360,121 @@ class FrevaGPT extends React.Component {
   }
 
   renderBotContent() {
+    const windowHeight = document.documentElement.clientHeight * 0.65;
+
+    // better solution needed (wasn't able to find any suitable bootstrap class -> need of fixed height for overflow-auto -> scrolling)
+    const chatWindow = {
+      height: windowHeight,
+    };
+
+    // here also no suitable solutions using bootstrap found -> need for better solution
+    const scrollButtonStyle = {
+      zIndex: 10,
+      right: "40px",
+      bottom: "10px",
+      position: "sticky",
+    };
+
     return (
       <>
-        <Col md={4}>
+        <Col md={3}>
           <SidePanel />
         </Col>
 
-        <Col md={8}>
-          {this.state.showSuggestions ? (
-            <Row className="mb-2 g-2">
-              {botSuggestions.map((element) => {
-                return (
-                  <div key={element} className="col-md-3">
-                    <OverlayTrigger
-                      key={element}
-                      overlay={<Tooltip>{element}</Tooltip>}
-                    >
-                      <Button
-                        className="h-100 w-100"
-                        variant="outline-secondary"
-                        onClick={() => this.handleSubmit(element)}
-                      >
-                        {truncate(element)}
-                      </Button>
-                    </OverlayTrigger>
-                  </div>
-                );
-              })}
-            </Row>
-          ) : null}
-
-          <ChatBlock />
-
-          <PendingAnswerComponent
-            content={this.state.dynamicAnswer}
-            variant={this.state.dynamicVariant}
-          />
+        <Col
+          md={9}
+          className={
+            "d-flex flex-column " +
+            (this.state.showSuggestions
+              ? "justify-content-start"
+              : "justify-content-between")
+          }
+          style={chatWindow}
+        >
+          <Row
+            className="overflow-auto position-relative"
+            id="chatContainer"
+            onScroll={debounce(this.handleScroll, 100)}
+          >
+            <Col md={12}>
+              <ChatBlock />
+
+              <PendingAnswerComponent
+                content={this.state.dynamicAnswer}
+                variant={this.state.dynamicVariant}
+              />
+
+              {this.state.loading && !this.state.dynamicAnswer ? (
+                <Row className="mb-3">
+                  <Col md={3}>
+                    <Card className="shadow-sm card-body border-0 border-bottom mb-3 bg-light d-flex flex-row align-items-center">
+                      <Spinner size="sm"/>
```

Comment:
At this point there is a quite wide variety of spinners (different sizes, different positions, sometimes with text). Just by looking at it from a user's perspective I find the changes during the bot's thought process rather confusing.